contracts/modules/legacy/LegacyPooledStaking.sol

```diff
@@ -1474,10 +1474,12 @@ contract LegacyPooledStaking is IPooledStaking, MasterAwareV2 {
       initialPoolFee,
       maxPoolFee,
       params,
-      deposit,
-      trancheIdInEffect,
       migrationData.ipfsDescriptionHash
     );
+
+    // TODO: create the first deposit
```

Comment:
another TODO left here.

Do we have a ticket to address this? Let's link it if so so we don't leave things hanging

Reply:
This function needs a bit of a rewrite as part of https://github.com/NexusMutual/smart-contracts/issues/608 - the TODO can be treated then.